apps/client/package.json

```diff
@@ -72,12 +77,13 @@
     "@types/react-router-dom": "^5.1.5",
     "batch-promises": "0.0.3",
     "fast-xml-parser": "^3.17.4",
-    "history": "^5.0.0",
```

Comment:
You somehow removed the "history" package and it's still imported in `LocationPage.test.js`. That's why the tests are failing. Can you re-add?

`npm i --save history@5.0.0`

Reply:
He that's weird, I must have overlooked. Thanks